app/javascript/__tests__/modules/listingDetails/ListingDetailsProcess.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import renderer from "react-test-renderer"
+import { ListingDetailsProcess } from "../../../modules/listingDetailsAside/ListingDetailsProcess"
+import { lotteryCompleteRentalListing } from "../../data/RailsRentalListing/listing-rental-lottery-complete"
+
+describe("ListingDetailsProcess", () => {
+  it("renders leasing agent section if exists", () => {
+    const tree = renderer
+      .create(<ListingDetailsProcess listing={lotteryCompleteRentalListing} />)
+      .toJSON()
+
+    expect(tree).toMatchSnapshot()
+  })
+})
```

Comment:
could be considered out of scope for this story, but eventually we'll want to add tests for the event section too